clickhouse_search/models.py

```diff
@@ -176,6 +424,48 @@ class Meta:
         engine = EmbeddedRocksDB(0, f'{CLICKHOUSE_DATA_DIR}/GRCh38/SNV_INDEL/annotations', primary_key='key', flatten_nested=0)
 
 
+class Clinvar(models.ClickhouseModel):
```

Comment:
this model had to be moved up in the file, its just copy-paste excet where noted